package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua

```diff
@@ -9,6 +9,9 @@ function M.section(form)
   if enabled then
     local s = form:section(cbi.SimpleSection, nil,
       i18n.translate('This node will automatically update its firmware when a new version is available.'))
+  else
+    local s = form:section(cbi.SimpleSection, nil,
+      i18n.translate("Automatic updates are disabled. They can be enabled in <em>Advanced Settings</em>"))
```

Comment:
Have you checked that HTML markup is working as expected in a section description?